### ISSUE TITLE
About boost::segment type on VS2019

### ISSUE THREAD

**Issue by Choy-Mutao** [awulkiew/graphical-debugging#52]
Its a wonderful application for Graphics programmer, thanks a lot for ur given;
is there no support boost::segment type on VS2019?

**Comment by awulkiew** [comment]
If by `boost::segment` you mean `boost::geometry::model::segment` then yes, they should work. User-defined segments are supported as well: https://github.com/awulkiew/graphical-debugging/blob/master/examples/cpp.xml#L131

**Comment by Choy-Mutao** [comment]
I'v tried some ways to show `boost::geometry::model::segment` again, however i failed;
```cpp
        boost::geometry::model::segment<boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian>> boost_segment;
        boost::geometry::model::linestring<boost::geometry::model::point<double, 2, boost::geometry::cs::cartesian>> boost_linestring;
        bg::set<0, 0>(boost_segment, 1.0);
        bg::set<0, 1>(boost_segment, 2.0);
        bg::set<1, 0>(boost_segment, 3.0);
        bg::set<1, 1>(boost_segment, 4.0);
        
        bg::append(boost_linestring, BoostPoint(0.0, 0.0));
        bg::append(boost_linestring, BoostPoint(1.0, 0.0));
        bg::append(boost_linestring, BoostPoint(1.0, 2.0));
```
On VS2019, except `boost::geometry::model::segment`, other is normal;

**Comment by awulkiew** [comment]
Fixed in v0.34.
